### PR TITLE
Add git remote

### PR DESCRIPTION
**Pull request by jorgeborges**
Adding git remote page.

### CODE REVIEW

File: pages/common/git-remote.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+- Show a list of existing remotes, their names and URL
+
+`git remote -v`
+
+- Add a remote
+
+`git remote add {{remote_name}} {{remote_url}}`
+
+- Add a remote and run fetch on it
```
Comment (waldyrious) [inline pages/common/git-remote.md]: is this really such a common use case? I'd suggest instead having the command to change the URL of a remote, which I personally have had to do more than once (e.g. after forgetting to make a clone of a github repo in the web interface before cloning to my machine, so I could push to it).

**Comment by jorgeborges** [comment]
I took your recommendation @waldyrious, most people would probably prefer to explicitly fetch anyway.

git remote set-url is harder to remember too, just added it.

**Comment by waldyrious** [comment]
:+1: :)

**Comment by rprieto** [comment]
It looks great, thanks for the contribution!

**Pull request merged**